src/main/java/com/rybka/service/ExchangeRateBaseCurrencyExchangeConnector.java

```diff
@@ -0,0 +1,22 @@
+package com.rybka.service;
+
+import com.rybka.CurrencyAPIConstants;
+import com.rybka.model.ExchangeResponse;
+import coresearch.cvurl.io.request.CVurl;
+import lombok.extern.log4j.Log4j;
+
+@Log4j
+public class ExchangeRateBaseCurrencyExchangeConnector implements BaseCurrencyExchangeConnector {
```

Comment:
I'd simplify this names to 
ExchangeRateConnector implements BaseCurrencyExchangeConnector
PrimeExchangeRateConnector implements BaseCurrencyExchangeConnector

Reply:
fixed